rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,435 @@
+
+#### Science Roadmap
+
+Oversight MUST provide an annual Science Roadmap to Project Leads to be reviewed at the next Oversight meeting following its receipt.
+
+Project Leads MUST clarify outstanding questions about the Science Roadmap with Oversight within two weeks.
+
+#### Technical Objectives RFC
+
+Project Leads MUST review the [Technical Objectives RFC](#technical-architecture-objectives) from the most recent quarter to understand outstanding technical concerns.
+
+#### DCP User Survey and UX Insights
+
+User Experience (UX) MUST create a DCP user survey on an annual basis to collect input for the Product Roadmap. The audience for the survey are the Users of the DCP.
+
+UX MUST review the proposed survey with the DCP community and incorporate suggestions as appropriate.
+
+The survey MUST be announced to the membership of the Human Cell Atlas (through its Registration mailing list) with a minimum of a two week last call for responses.
+
+At the conclusion, UX MUST summarize insights from the survey and present at the next DCP PM meeting. Following a one week review to incorporate feedback, the survey MUST be shared by UX with the membership of the Human Cell Atlas and announced on the *HumanCellAtlas #dcp* slack channel.
+
+UX MAY also share critical UX Insights from other sources independent of the Survey for consideration by the Project Leads during the creation of the Roadmap.
+
+#### Roadmap Retrospective
+
+The Roadmap Retrospective occurs during the recurring Project Leads meeting and not in a special session. The Roadmap Shepherd MUST add the Retrospective to the agenda and faciliate.
+
+In preparation, the Project Leads MUST independently review the prior year’s Product Roadmap, Release Retrospective summaries, and appropriate supporting documentation and then share their significant findings and learnings with their peers during the Retrospective.
+
+Findings may include: Themes that were incomplete and the accompanying reason, incorrectly sized work in the Engineering Plan, etc. These findings SHOULD be utilized as a reference class for the next Product Roadmap.
+
+Project Leads MUST identify a Roadmap Shepherd during the Retrospective. The Shepherd MUST NOT be a Project Lead, to ensure the neutrality of the content for the Roadmap.
+
+The Shepherd is responsible for:
+* Managing the timeline for the creation of the draft Roadmap
+* Facilitating all meetings related to the creation of the draft Roadmap 
+* Documenting important decisions, discussions, and questions 
+* Assigning action items to move toward the creation of the Roadmap, should no Project Lead volunteer, and following up with them through completion
+
+### Drafting the Roadmap
+[Drafting the Roadmap]:#drafting-the-roadmap
+
+Project Leads MUST independently create Themes by utilizing the input provided in the Science Roadmap, Technical Objectives RFC, DCP User Survey and UX Insights, and Roadmap Retrospective.
```

Comment:
...and the Aims from the DCP Strategy doc

Reply:
I like the spirit of your comment @jahilton, and would phrase it like:

"Project Leads MUST independently create Themes by utilizing the input provided in the Science Roadmap, Technical Objectives RFC, DCP User Survey and UX Insights, and Roadmap Retrospective and align those themes to aim(s) from the DCP Strategic aims document."

What are your thoughts about this?

Reply:
Are all reviewers familiar with the the [Human Cell Atlas DCP: Strategy](https://docs.google.com/document/d/1hBsj2BB9SjuJHodS3hM7cRJzXc5x-uYdOQL3LtGjYWM/edit?ts=5d24fa90#heading=h.t9hisgss8no3) document? I know that it's been iterating with Project Leads and Oversight, but has there been broader awareness? I'm also hesitant to reference until it is finalized and published in a public location. Is this going to be an Informational RFC or portal content? Will this document be updated?

Reply:
@tburdett - can you comment on my questions related to the Strategy document?